### PR TITLE
Fix typo error

### PR DESCRIPTION
**Pull request by ccbond**
- fix typo.
 - delete useless import.

### CODE REVIEW

**Comment by ccbond** [comment]
@microsoft-github-policy-service agree

**Comment by ccbond** [comment]
> @ccbond please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by hjonasson** [comment]
Perhaps the real fix is to add eslint (or some other linter) 😉

File: src/validate.ts [hunk 1]
```diff
@@ -76,8 +76,7 @@ export function createJsonValidator<T extends object = object>(schema: string, t
         let jsonObject;
         try {
             jsonObject = JSON.parse(jsonText) as object;
-        }
-        catch (e) {
+        } catch (e) {
```
Comment (DanielRosenwasser) [inline src/validate.ts:79]:
```suggestion
        }
        catch (e) {
```

**Comment by DanielRosenwasser** [comment]
Probably! We'll probably add dprint [with the same config as we use in TypeScript](https://github.com/microsoft/TypeScript/blob/80ab111685e3e0c10d8d13cdd18d8dffa8328895/.dprint.jsonc).

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**